drivers/shared/executor/executor_linux_test.go

```diff
@@ -494,7 +495,6 @@ CapAmb: 0000000000000000`,
 
 	for _, c := range cases {
 		t.Run(c.user, func(t *testing.T) {
-			require := require.New(t)
```

Comment:
This delights me - I dislike `require.New(t)` - it doesn't really save any keystores and makes it easy to get subtest assertions associated with the wrong `t`.